src/features/callAssignments/components/StatusSectionHeader.tsx

```diff
@@ -0,0 +1,53 @@
+import { makeStyles } from '@material-ui/styles';
+import { Box, Divider, Theme, Typography } from '@material-ui/core';
+
+interface StatusSectionHeaderProps {
+  chipColor: string;
+  subtitle: string;
+  targetingDone: boolean;
+  title: string;
+  value: number;
+}
+
+const useStyles = makeStyles<Theme, { chipColor: string }>(() => ({
+  chip: {
+    backgroundColor: ({ chipColor }) => chipColor,
+    borderRadius: '1em',
+    color: 'white',
+    display: 'flex',
+    fontSize: '1.8em',
+    lineHeight: 'normal',
+    marginRight: '0.1em',
+    overflow: 'hidden',
+    padding: '0.2em 0.7em',
+  },
+}));
+
+const StatusSectionHeader = ({
+  chipColor,
+  subtitle,
+  targetingDone,
```

Comment:
Not sure how I feel about this boolean, compared to just passing `undefined` to the value. Do you like this better?